app/components/Poll/index.js

```diff
@@ -138,16 +147,18 @@ class Poll extends React.Component<Props, State> {
           </Flex>
         )}
         <div style={{ height: '29px' }}>
-          {truncateOptions && (
-            <div className={styles.moreOptionsLink}>
+          <div className={styles.moreOptionsLink}>
+            <span style={{ width: '40%' }}>{`Stemmer: ${totalVotes}`}</span>
+            {truncateOptions && (
               <Icon
                 onClick={this.toggleTruncate}
                 className={styles.arrow}
                 size={20}
                 name={expanded ? 'arrow-up' : 'arrow-down'}
               />
-            </div>
-          )}
+            )}
+            <span style={{ width: '40%' }} />
```

Comment:
Nah m8, empty span with a width?? By default, span element is an inline element, and it ignores width. You can display it as `inline-block` if you really want this effect.